### ISSUE TITLE
limel-checkbox: there is no focus indication for keyboard navigation

### ISSUE THREAD

**Issue by adrianschmidt** [Lundalogik/lime-elements#564]
## Current behavior

When tabbing to a limel-checkbox, there is absolutely no indication that the checkbox is focused.

## Expected behavior

There should be some indication that the checkbox is focused. It's ok if it's not visible when clicking or tapping the checkbox, but when navigating with the keyboard, there has to be some visual indication of focus.

There is most likely some implementation of this in MDC already, so we've probably just missed adding some ripple effect or something.

## Environment

- **lime-elements** version: 26.1.1
- Framework used: Can be reproduced in the docz app.
